stdplugins/quotly.py

```diff
@@ -19,26 +19,25 @@ async def _(event):
         quote = reply
     else:
         return
-    bot = "@QuotLyBot"
-    await event.edit("```Making a quote....```")
-
-    async with borg.conversation(bot) as bot_conv:
-        if True:  # lazy indentation workaround xD
+    username = "@QuotLyBot"
+    await event.edit(f"```Quoting this message...```")
+    async with client.conversation(username, timeout=10) as bot_conv:
+        try:
             if input_str:
                 response = await silently_send_message(bot_conv, quote)
             elif reply:
-                response = bot_conv.wait_event(
-                    events.NewMessage(
-                        incoming=True,
-                        from_users=1031952739))
-                await borg.forward_messages(bot, quote)
+                response = bot_conv.wait_event(telethon.events.NewMessage(
+                    incoming=True, from_users=1031952739))
+                await client.forward_messages(username, quote)
                 response = await response
                 response = response.message
             if response.text.startswith("Command"):
-                await event.edit("Invalid message type.")
+                await event.edit(f"Invalid message type.")
```

Comment:
Codacy found an issue: [f-string is missing placeholders (F541)](https://app.codacy.com/manual/prono69/PepeBot/pullRequest?prid=6191017)